file/file_util.cpp

```diff
@@ -25,7 +25,7 @@
 #include "file/file_util.h"
 #include "util/text/utf8.h"
 
-#if defined(__FreeBSD__) || defined(__APPLE__)
+#if !defined(__linux__) && !defined(__SYMBIAN32__)
```

Comment:
It's such a small change, I went ahead and just cherry picked it: 5984a4aa1cee5f66370902296346be5c91d3a628

-[Unknown]